libdnf/hy-goal.c

```diff
@@ -1074,14 +1131,54 @@ hy_goal_describe_problem(HyGoal goal, unsigned i)
         }
         return g_strdup(string->str);
     }
+    pset = hy_goal_broken_dependency_all_pkgs(goal, DNF_PACKAGE_STATE_INSTALLED);
+    const int count = dnf_packageset_count(pset);
```

Comment:
@ignatenkobrain I think this requested change doesn't follow aim of pull-request, therefore it shoul be part of another work. The rest was fixed.